lib/rt-mailbox/class-rt-zend-mail.php

```diff
@@ -274,7 +274,16 @@ public function sendemail( $fromname, $fromemail, $accessToken, $email_type, $im
 			$transport->setOptions( $options );
 
 			$message = new Message();
-			$message->addFrom( $fromemail, $fromname );
+			
+			/**
+			 * Filter the from email address.
+			 * @param string $fromemail Actual from email address.
+			 * @param int $post_id Ticket ID.
+			 */
+			$new_fromemail = apply_filters( 'rtlib_from_header', $fromemail, $post_id );
```

Comment:
We get `$post_id` on line 294 or 297 and you're using it on 283 before initializing?